recipe/conda_build_config.yaml

```diff
@@ -1,4 +1,6 @@
-MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
-  - 11.0                   # [osx and x86_64]
-MACOSX_SDK_VERSION:        # [osx and x86_64]
-  - 11.0                   # [osx and x86_64]
+c_stdlib:
+  - sysroot                    # [linux]
+  - macosx_deployment_target   # [osx]
```

Comment:
It doesn't exactly "hurt", but overriding `c_stdlib` is unnecessary. You only need to set the version, not the _name_ of the metapackage.

Reply:
Ah ok that's good to know! I just blindly copy-pasted the example in the description of the linked issue https://github.com/conda-forge/conda-forge.github.io/issues/2102 .